flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -380,6 +389,53 @@ public void testTableSourceCommitOnCheckpointDisabled() {
                                 .noDefaultValue()));
     }
 
+    @ParameterizedTest
```

Comment:
Can you also quickly migrate all the other tests to use junit 5 because mixing them in the same class causes problems? It would be nice to do it as a separate commit.

Reply:
fixed